bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -177,6 +178,16 @@ Detects every movement through an intelligent combination of passive infra-red t
 | battery-level   | Number    | &#9744;  | Current battery level percentage as integer number. Bosch-specific battery levels are mapped to numbers as follows: `OK`: 100, `LOW_BATTERY`: 10, `CRITICAL_LOW`: 1, `CRITICALLY_LOW_BATTERY`: 1, `NOT_AVAILABLE`: `UNDEF`. |
 | low-battery     | Switch    | &#9744;  | Indicates whether the battery is low (`ON`) or OK (`OFF`). |
 
+### Presence Simulation
+
+Simulates that someone is home.
+
+**Thing Type ID**: `presence-simulation`
+
+| Channel Type ID             | Item Type | Writable | Description                                  |
```

Comment:
```suggestion
| Channel ID                  | Item Type | Writable | Description                                  |
```

Reply:
Yes, I agree this should be changed everywhere. Are you ok if we merge it like that, and then I create a separate PR that fixes this issue everywhere? Then I can also apply another enhancement you proposed, where we could replace all those check mark symbols in the `Writable` columns with something like `yes`/`no`. Btw, is there any guideline how those channel documentation tables should look like? Or can you recommend a binding documentation that could be used as template?

Reply:
That’s ok. There is no detailed rule set for the tables, atleast not known to me.
Most bindings use a very similar set of columns: Id, description, default value, advanced, required.